pkg/prometheus_metrics/migration.go

```diff
@@ -0,0 +1,93 @@
+package metrics
+
+import (
+	"fmt"
+
+	stork_api "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// migrationStatusCounter for migration status
+	migrationStatusCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "migration_status",
+		Help: "Status of migration",
+	}, []string{MetricName, MetricNamespace, MetricSchedule})
+	// migrationStageCounter for migration CR stages on server
+	migrationStageCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "migration_stage",
+		Help: "Stage of migration",
+	}, []string{MetricName, MetricNamespace, MetricSchedule})
+	// migrationDurationCounter for time taken by migration to complete
+	migrationDurationCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "migration_duration",
+		Help: "Duration of migrations",
+	}, []string{MetricName, MetricNamespace, MetricSchedule})
+)
+
+var (
+	// MigrationStatus map of application migration status to enum
+	MigrationStatus = map[stork_api.MigrationStatusType]float64{
```

Comment:
```suggestion
	migrationStatus = map[stork_api.MigrationStatusType]float64{
```
Same for the other maps